conda/recipes/libraft/meta.yaml

```diff
@@ -40,21 +40,39 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
         - librmm
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }} {{ cuda_version }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }} {{ cuda_version }}
+        {% else %}
+        - {{ compiler('cuda') }}
+        {% endif %}
+        - cuda-version ={{ cuda_version }}
         - cmake {{ cmake_version }}
         - ninja
         - sysroot_{{ target_platform }} {{ sysroot_version }}
       host:
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - libnvjitlink
+        {% endif %}
+        - cuda-version ={{ cuda_version }}
         - librmm ={{ minor_version }}
-        - cudatoolkit {{ cuda_version }}
       run:
-        - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - cuda-cudart
+        - libnvjitlink
```

Comment:
Can we try putting the `-dev` variant of these packages into `host` and remove these from `run`? I'm trying the same in cuspatial: [rapidsai/cuspatial@`cbd9ac1` (#1044)](https://github.com/rapidsai/cuspatial/pull/1044/commits/cbd9ac1832db4f7c700c27db1107baee5fc8935b). Using `run_exports` plays better with the conda overlinking checker and results in fewer warnings.